connect.go

```diff
@@ -0,0 +1,471 @@
+		subMsg := "Press 'R' to Restart"
+		msgX := (connectScreenWidth - len(msg)*7) / 2
+		subMsgX := (connectScreenWidth - len(subMsg)*7) / 2
+		text.Draw(screen, msg, basicfont.Face7x13, msgX, connectScreenHeight/2-20, color.Red)
+		text.Draw(screen, subMsg, basicfont.Face7x13, subMsgX, connectScreenHeight/2, color.Black)
+	}
+}
+
+func (g *ConnectGame) drawTile(screen *ebiten.Image, tile *Tile, x, y int) {
+	tileImg := ebiten.NewImage(tileSize-2, tileSize-2)
+	tileColor := tileColors[tile.Type]
+	
+	if tile.Selected {
+		// Add selection highlight
+		tileColor = color.RGBA{255, 255, 255, 255}
+	}
+	
+	tileImg.Fill(tileColor)
+	
+	op := &ebiten.DrawImageOptions{}
+	op.GeoM.Translate(float64(boardOffsetX+x*tileSize+1), float64(boardOffsetY+y*tileSize+1))
+	screen.DrawImage(tileImg, op)
+	
+	// Draw tile type indicator (simple text for now)
+	var typeChar string
+	switch tile.Type {
+	case TileAnimal1:
+		typeChar = "🐱"
+	case TileAnimal2:
+		typeChar = "🐶"
+	case TileAnimal3:
+		typeChar = "🐼"
+	case TileAnimal4:
+		typeChar = "🐯"
+	case TileFruit1:
+		typeChar = "🍎"
+	case TileFruit2:
+		typeChar = "🍊"
+	case TileFruit3:
+		typeChar = "🍓"
+	case TileFruit4:
+		typeChar = "🍇"
+	}
+	
+	if typeChar != "" {
+		textX := boardOffsetX + x*tileSize + 8
+		textY := boardOffsetY + y*tileSize + 20
+		text.Draw(screen, typeChar, basicfont.Face7x13, textX, textY, color.White)
+	}
+}
+
+func (g *ConnectGame) drawPath(screen *ebiten.Image) {
+	if len(g.connectionPath) < 2 {
+		return
+	}
+
+	// Draw lines connecting the path points
+	for i := 0; i < len(g.connectionPath)-1; i++ {
+		p1 := g.connectionPath[i]
+		p2 := g.connectionPath[i+1]
+		
+		// Convert board coordinates to screen coordinates
+		x1 := boardOffsetX + (p1.X-1)*tileSize + tileSize/2
+		y1 := boardOffsetY + (p1.Y-1)*tileSize + tileSize/2
+		x2 := boardOffsetX + (p2.X-1)*tileSize + tileSize/2
+		y2 := boardOffsetY + (p2.Y-1)*tileSize + tileSize/2
+		
+		g.drawLine(screen, x1, y1, x2, y2, color.RGBA{255, 255, 0, 255})
+	}
+}
+
+func (g *ConnectGame) drawLine(screen *ebiten.Image, x1, y1, x2, y2 int, c color.Color) {
+	// Simple line drawing - draw a thick line using rectangles
+	if x1 == x2 {
+		// Vertical line
+		minY, maxY := y1, y2
+		if y1 > y2 {
+			minY, maxY = y2, y1
+		}
+		lineImg := ebiten.NewImage(4, maxY-minY)
+		lineImg.Fill(c)
+		op := &ebiten.DrawImageOptions{}
+		op.GeoM.Translate(float64(x1-2), float64(minY))
+		screen.DrawImage(lineImg, op)
+	} else if y1 == y2 {
+		// Horizontal line
+		minX, maxX := x1, x2
+		if x1 > x2 {
+			minX, maxX = x2, x1
+		}
+		lineImg := ebiten.NewImage(maxX-minX, 4)
+		lineImg.Fill(c)
+		op := &ebiten.DrawImageOptions{}
+		op.GeoM.Translate(float64(minX), float64(y1-2))
+		screen.DrawImage(lineImg, op)
+	}
+}
+
+func (g *ConnectGame) Layout(outsideWidth, outsideHeight int) (int, int) {
+	return connectScreenWidth, connectScreenHeight
+}
+
+// ConnectMain runs the connect game
+func ConnectMain() {
+	rand.Seed(time.Now().UnixNano())
```

Comment:
Remove deprecated `rand.Seed()` call. Go 1.20+ uses auto-seeding by default, and the current approach conflicts with main.go which also seeds the random generator. The math/rand package now provides better default seeding automatically.

```suggestion
func ConnectMain() {
	ebiten.SetWindowSize(connectScreenWidth, connectScreenHeight)
	ebiten.SetWindowTitle("Connect Game - 连连看")
	if err := ebiten.RunGame(NewConnectGame()); err != nil {
		log.Fatal(err)
	}
}
```

Reply:
🤖 CodeAgent is working… 

I'll analyze this and get back to you.

Reply:
@qiniu-ci 那么我该怎么做你下次才能不会出现这样的错误，我需要在给你的提示词补充什么说明或者约束吗

Reply:
🤖 CodeAgent is working… 

I'll analyze this and get back to you.